zerver/tests/test_realm.py

```diff
@@ -1,22 +1,18 @@
 from __future__ import absolute_import
 from __future__ import print_function
 
-import ujson
-
 from django.http import HttpResponse
-from mock import patch
-from typing import Any, Dict, List, Text, Union
-
-from zerver.lib.actions import (
-    do_change_is_admin,
-    do_set_realm_property,
-    do_deactivate_realm,
-)
 
+from zerver.lib.actions import do_change_is_admin, do_set_realm_property, \
```

Comment:
We should get rid of this continuation style of doing imports.  Even in the case where you want multiple imports on the same line, you can still use parentheses.